fix: update grpc_naming.md for using etcd discovery with go-grpc code…

Aaron-cdx opened the pull request.

jmhbnz reviewed (changes requested):
Hi @Aaron-cdx - Thanks for raising this - I notice you have proposed updating the documentation for version `3.3`.  This version is no longer supported or maintained so the documentation is frozen.  

Can you please check the documentation for versions `3.4`, `3.5` and `3.6` and update those if needed?  Those versions are what is currently supported and maintained.

Aaron-cdx commented:
hi @jmhbnz, thanks for your reminder, I removed the changes for 3.3 and checked and added the changes for `3.4 & 3.5 & 3.6`.

jmhbnz commented:
> hi @jmhbnz, thanks for your reminder, I removed the changes for 3.3 and checked and added the changes for `3.4 & 3.5 & 3.6`.

Thanks - Can you please ensure your commit is signed so the Developer Certificate of Origin check passes? i.e


```bash
git reset --soft HEAD~1
git commit --signoff -m "<message>"
git push --force
```

Aaron-cdx commented:
> > hi @jmhbnz, thanks for your reminder, I removed the changes for 3.3 and checked and added the changes for `3.4 & 3.5 & 3.6`.
> 
> Thanks - Can you please ensure your commit is signed so the Developer Certificate of Origin check passes? i.e
> 
> ```shell
> git reset --soft HEAD~1
> git commit --signoff -m "<message>"
> git push --force
> ```

Hi @jmhbnz completed, please help to check, thank you!

jmhbnz reviewed (approved):
LGTM - Thanks for tidying this up @Aaron-cdx

ivanvc commented on content/en/docs/v3.5/dev-guide/grpc_naming.md:
````diff
@@ -12,15 +12,23 @@ The etcd client provides a gRPC resolver for resolving gRPC endpoints with an et
 
 ```go
 import (
-	clientv3 "go.etcd.io/etcd/client/v3"
````
This may be nice to keep because the module is named "clientv3" and not "v3"; therefore, `goimports` sets it as a named import with "clientv3". So, the suggestion is to keep this line and also for v3.6.

Aaron-cdx replied:
got it, revert it to fixed, thank you @ivanvc!

ivanvc reviewed (changes requested):
Thanks, @Aaron-cdx. This is looking great. However, the imports from v3.4 need to be updated. v3.5 and 3.6 work, but I left a suggestion.

ivanvc reviewed (approved):
LGTM. It looks like you reverted v3.4's changes, which needs to be updated too. But we can do it in a follow-up pull request.
